tests/tas-cmis/pom.xml

```diff
@@ -67,6 +67,12 @@
             <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.epam.reportportal</groupId>
+            <artifactId>agent-java-testng</artifactId>
+            <version>5.4.0</version>
```

Comment:
The version should be unnecessary, it should already be handled via the `dependencyManagement` of the parent POM. Can we test removing it?